board/model_manager.py

```diff
@@ -118,14 +131,63 @@ def sample_data_and_train(self):
         self.logger.info("############ instantiating Recommender.. ############")
 
         
-
-        bpr_model = BPR(batch_size=1000, 
-                        total_users=train_dataset.total_users(), 
-                        total_items=train_dataset.total_items(), 
-                        dim_user_embed=50, 
-                        dim_item_embed=50, 
-                        save_model_dir='data/'+str(self.model_id)+'/', 
-                        train=True, serve=True)
+        model_save_dir = self.recommender_name+'_recommender/'
+        if self.recommender_name=="bpr":
+            print("recommender chosen is:", self.recommender_name)
```

Comment:
These comments will be removed post completion for model_algorithm-evaluator-sampler validation in next PR